dimagi/pages/views/toolkits.py

```diff
@@ -23,3 +23,9 @@ def view_single(request, slug):
         'toolkit': toolkit,
     }
     return render(request, 'pages/toolkits/view_single.html', context)
+
+
+def temporary_redirect_toolkits(request):
```

Comment:
this is a lot of duplicated code. can you make a new temporary redirect function that takes in an argument and replaces the status code? We might need to use this again.